### DISCUSSION TITLE
Ability to view logs?

### DISCUSSION THREAD

**Discussion by prescience-data** [supabase/supabase#1803] [Questions] [answered]
Is there currently a way to view error logs in the supabase.io GUI?

I have a function that is failing to insert but the message returned to the client isn't helpful as it only explains the return type is not a match (due to no rows being inserted), so trying to work out how to debug it.

**Comment by inian** [comment]
Hi @prescience-data, this is something that we are working on right now - showing metrics and logs in the dashboard. In the mean time, if you give your project ref (or email to beta@supabase.io), happy to manually check the database for any error logs.

**Comment by Ziinc** [comment, marked as answer]
hi @prescience-data ,additional logging features have been made available in the Dashboard, and documentation on its features and capabilities are available [here](https://supabase.com/docs/guides/platform/logs).
